buzzfeedbotOS.py

```diff
@@ -115,38 +97,37 @@ def clickbait_meat(link_to_check, total_points):
 	for title in soup.find_all('h3'):
 		for number in title.find_all('span', attrs={'class': 'subbuzz__number'}):
 			bullet_point = True
-		if bullet_point == True or any_numbered_points == False:
+		if bullet_point or not any_numbered_points:
 			for article in title.find_all('span', attrs={'class': 'js-subbuzz__title-text'}):
 				if len(article.text)<4 or article.text.endswith(':'):
 					return ''

```

Comment:
When the `if` part is just a return and the rest of the code is in the `else` part, you can just omit the `else` and avoid the extra indentation.